{
  "raw_data": [
    {
      "_cyevm": "Mahe",
      "agebracket": "68",
      "backupnotes": "Travel history UAE",
      "contractedfromwhichpatientsuspected": "",
      "currentstatus": "Hospitalized",
      "dateannounced": "17/03/2020",
      "detectedcity": "Mahe",
      "detecteddistrict": "Mahe",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "F",
      "nationality": "India",
      "notes": "Travelled from UAE",
      "patientnumber": "140",
      "source1": "https://twitter.com/PTI_News/status/1239866086905921536?s=19",
      "source2": "https://www.thehindu.com/news/national/kerala/first-covid-19-case-detected-in-mahe/article31093347.ece",
      "source3": "https://www.facebook.com/CollectorKKD/posts/2591627614407336",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "17/03/2020",
      "typeoftransmission": "Imported"
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "58",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "E0",
      "currentstatus": "Hospitalized",
      "dateannounced": "01/04/2020",
      "detectedcity": "Ariyankuppam",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "M",
      "nationality": "India",
      "notes": "Travelled to Delhi",
      "patientnumber": "1659",
      "source1": "https://www.newindianexpress.com/states/tamil-nadu/2020/apr/01/two-from-puducherry-who-attended-nizamuddin-congregation-test-positive-for-covid-19-2124270.html",
      "source2": "https://twitter.com/timesofindia/status/1245201501594644480",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "01/04/2020",
      "typeoftransmission": "Local"
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "31",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "E0",
      "currentstatus": "Hospitalized",
      "dateannounced": "01/04/2020",
      "detectedcity": "Ariyankuppam",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "M",
      "nationality": "India",
      "notes": "Travelled to Delhi",
      "patientnumber": "1660",
      "source1": "https://www.newindianexpress.com/states/tamil-nadu/2020/apr/01/two-from-puducherry-who-attended-nizamuddin-congregation-test-positive-for-covid-19-2124270.html",
      "source2": "https://twitter.com/timesofindia/status/1245201501594644480",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "01/04/2020",
      "typeoftransmission": "Local"
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "",
      "currentstatus": "Hospitalized",
      "dateannounced": "02/04/2020",
      "detectedcity": "Ariyankuppam",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "",
      "nationality": "",
      "notes": "",
      "patientnumber": "2542",
      "source1": "https://www.jipmer.edu.in/novel-corona-virus",
      "source2": "https://www.newindianexpress.com/states/tamil-nadu/2020/apr/02/kin-of-covid-19-patient-who-went-to-tablighi-meet-in-delhi-tests-positive-in-puducherry-2124897.html",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "02/04/2020",
      "typeoftransmission": ""
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "",
      "currentstatus": "Hospitalized",
      "dateannounced": "02/04/2020",
      "detectedcity": "Thiruvandarkoil",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "",
      "nationality": "",
      "notes": "",
      "patientnumber": "2543",
      "source1": "https://www.jipmer.edu.in/novel-corona-virus",
      "source2": "https://timesofindia.indiatimes.com/city/puducherry/covid-19-two-more-test-positive-in-puducherry/articleshow/74953118.cms",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "02/04/2020",
      "typeoftransmission": ""
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "47",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "E0",
      "currentstatus": "Hospitalized",
      "dateannounced": "10/04/2020",
      "detectedcity": "Moolakulam",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "",
      "nationality": "India",
      "notes": "Delhi Conference",
      "patientnumber": "6911",
      "source1": "https://twitter.com/PTI_News/status/1248555682351046656",
      "source2": "https://collectorate.py.gov.in/Corona%20COVID-19.htm",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "10/04/2020",
      "typeoftransmission": "Local"
    },
    {
      "_cyevm": "Puducherry",
      "agebracket": "54",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "E0",
      "currentstatus": "Hospitalized",
      "dateannounced": "10/04/2020",
      "detectedcity": "Thiruvandarkoil",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "",
      "nationality": "India",
      "notes": "Delhi Conference",
      "patientnumber": "6912",
      "source1": "https://twitter.com/PTI_News/status/1248555682351046656",
      "source2": "https://collectorate.py.gov.in/Corona%20COVID-19.htm",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "10/04/2020",
      "typeoftransmission": "Local"
    },
    {
      "agebracket": "18",
      "backupnotes": "",
      "contractedfromwhichpatientsuspected": "",
      "currentstatus": "Hospitalized",
      "dateannounced": "25/04/2020",
      "detectedcity": "",
      "detecteddistrict": "Puducherry",
      "detectedstate": "Puducherry",
      "estimatedonsetdate": "",
      "gender": "M",
      "nationality": "",
      "notes": "Son of a positive patient",
      "patientnumber": "24841",
      "source1": "https://www.thehindu.com/news/cities/puducherry/covid-19-one-more-tests-positive-in-puducherry/article31431309.ece",
      "source2": "",
      "source3": "",
      "statecode": "PY",
      "statepatientnumber": "",
      "statuschangedate": "25/04/2020",
      "typeoftransmission": ""
    }
  ]
}